domain/infosync/info.go

```diff
@@ -1047,3 +1057,29 @@ func ConfigureTiFlashPDForPartitions(accel bool, definitions *[]model.PartitionD
 	}
 	return nil
 }
+
+// StoreInternalSession is the entry function for store an internal session to SessionManager.
+func StoreInternalSession(se interface{}) {
+	is, err := getGlobalInfoSyncer()
+	if err != nil {
+		return
+	}
+	sm := is.GetSessionManager()
+	if sm == nil {
+		return
+	}
+	sm.StoreInternalSession(se)
+}
+
+// DeleteInternalSession is the entry function for delete an internal session from SessionManager.
+func DeleteInternalSession(se interface{}) {
+	is, err := getGlobalInfoSyncer()
+	if err != nil {
+		return
+	}
+	sm := is.GetSessionManager()
+	if sm == nil {
+		return
+	}
+	sm.DeleteInternalSession(se)
+}
```

Comment:
Can you explain why we need these two functions defined in `infosync`? Now, I search the code and only find them used from `session`, but `session` itself has `GetSessionManager`. So, I think it's already convenient to write `s.GetSessionManager().StoreInternalSession(se)` (probably need a nil check)

Reply:
The initialization process of the internal and user sessions are quite different, it seems the refactoring work is paused by now. Maybe we could file an issue to record this.

Reply:
I will do it